integration-tests/master-openshift/pom.xml

```diff
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.camel.quarkus</groupId>
+        <artifactId>camel-quarkus-build-parent-it</artifactId>
+        <version>2.13.0-SNAPSHOT</version>
+        <relativePath>../../poms/build-parent-it/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>camel-quarkus-integration-test-master-openshift</artifactId>
+    <name>Camel Quarkus :: Integration Tests :: Master With OpenShift</name>
+    <description>Integration tests for Camel Quarkus Master extension with OpenShift</description>
+
+    <properties>
+        <quarkus.runner>${project.build.directory}/quarkus-app/quarkus-run.jar</quarkus.runner>
+        <enforcer.skip>true</enforcer.skip>
```

Comment:
@jamesnetherton @ppalaga The `quarkus-test-openshift-client` provides openshift mocking feature. However, it uses junit4 which is banned. I've worked around with `enforcer.skip=true`.

Is it safe to include this test which skip enforcer ?

Reply:
I think we had a similar issue with other dependencies. Here's how we worked around it:

https://github.com/apache/camel-quarkus/blob/main/integration-tests/activemq/pom.xml#L57-L72

Maybe it could work here too.

Reply:
And it works like a charm in this case also. Many thanks for the tip @jamesnetherton :+1: